src/anndata/_core/merge.py

```diff
@@ -705,7 +705,10 @@ def _apply_to_awkward(self, el: AwkArray, *, axis, fill_value=None):
         else:
             if len(self.new_idx) > len(self.old_idx):
                 el = ak.pad_none(el, 1, axis=axis)  # axis == 0
-            return el[self.old_idx.get_indexer(self.new_idx)]
+            return el[self.get_new_idx_from_old_idx()]
+
+    def get_new_idx_from_old_idx(self):
```

Comment:
hm, is that the correct name? wouldn’t the accurate name contain an `into` or do I understand this API wrong (pandas docs are down, so I can‘t check)

Reply:
I'm not sure about `into` but perhaps just `indexer` and make this a `property`